java-client/flight-examples/src/main/java/io/deephaven/client/examples/DoPutSpray.java

```diff
@@ -48,6 +50,7 @@ public Void call() throws Exception {
 
         final Factory sourceFactory = FlightSessionFactoryConfig.builder()
                 .clientConfig(source.config())
+                .clientChannelFactory(CLIENT_CHANNEL_FACTORY)
```

Comment:
to confirm, when this is called, it will override the `CLIENT_CHANNEL_FACTORY` (semi-confusingly named) field in BarrageSessionFactoryConfig, which supplied the `deephaven-java-client-barrage` string, so we'll only have `deephaven-examples` instead, rather than in addition to?

Reply:
Correct, in addition to the version info:

default:
```
grpc-java/1.58.0 (deephaven/0.36.0-SNAPSHOT; flight/13.0.0; deephaven-java-client-flight)
```

as overridden in flight-examples:
```
grpc-java/1.58.0 (deephaven/0.36.0-SNAPSHOT; flight/13.0.0; deephaven-examples)
```

I do see I've re-used the same name throughout the examples, so I will switch them up. This instance will now be `deephaven-flight-examples`.